playbook/app/pb_kits/playbook/pb_date_picker/plugins/quickPick.tsx

```diff
@@ -11,6 +11,8 @@ type FpTypes = {
     append: (arg0: HTMLDivElement) => void;
   },
   loadedPlugins: string[],
+  // Add custom property to store quickpick label
+  _quickpickLabel?: string,
```

Comment:
Will this work with the custom version of the quickpick?  Where the dev chooses the options in the dropdown?